HIP/hip-0000-cancun-1.md

```diff
@@ -0,0 +1,115 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Add EVM Support for new non-blob Cancun opcodes
+author: Danno Ferrin (@shemnon)
+working-group: // TBD
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: // TBD
+discussions-to: // TBD
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+Update the Hedera EVM to add support for new Opcodes found in the Cancun fork of
+Ethereum Mainnet, namely TSTORE, TLOAD, and MCOPY
+
+## Motivation
+
+Hedera's goal of Ethereum Equivalence also includes the requirement to stay
+up-to-date with the current state of Ethereum Mainnet. This HIP addresses new
+opcodes added to the EVM that do not address any features relating to Blobs, or
+the Consensus Layer, or changes to existing opcodes.
+
+## Rationale
+
+The opcodes discussed in this HIP only exist within the EVM and do not interact
+with any novel features of ethereum mainnet. It is expected that future versions
+of solidity will support both implicit and explicit use of these opcodes.
+Because of that we need to support them as specified.
+
+## User stories
+
+* As a smart contract author, I want to be able to use transient storage
+  features of solidity in my hedera smart contracts.
+* As a smart contract author, I want to be able to use future versions of
+  solidity that may use memory copying implicitly, like what happened with
+  the `PUSH0` operation.
+* As a smart contract author, I want to be able to explicitly use the memory
+  copy features solidity may expose in future releases.
+
+## Specification
+
+### EVM Implementation
+
+Two EIPs define the operational semantics of the added opcodes. For the
+transient storage opcodes `TSTORE` and `TLOAD` they are defined
+in [EIP-1153](https://eips.ethereum.org/EIPS/eip-1153). For the `MCOPY`
+operation they are defined
+in [EIP-5656](https://eips.ethereum.org/EIPS/eip-5656).
+
+The specified opcodes are to be implemented identically to Ethereum Mainnet and
+as specified in their respective EIPs. This includes opcode numbers, gas
+schedules, stack semantics, and new facilities such as transient storage added
+to the execution frame.
+
+### Hedera Activation
+
+The operations will be added into a new EVM version of Hedera (notionally
+version 0.50, but subject to change), like the versions added for Shanghai
```

Comment:
Drop this section.
Once the HIP is implemented the doc will be updated with the correct services version